WalletWasabi.Fluent/Models/Wallets/WalletCoinjoinModel.cs

```diff
@@ -62,17 +65,23 @@ public WalletCoinjoinModel(Wallet wallet, IWalletSettingsModel settings)
 				.Select(_ => false);
 
 		IsRunning =
-			coinjoinStarted.Merge(coinjoinStopped)
+			coinjoinInputStarted.Merge(coinjoinStopped)
 				.Merge(coinjoinCompleted)
 						   .ObserveOn(RxApp.MainThreadScheduler);
 
 		IsRunning.BindTo(this, x => x.IsCoinjoining);
+
+		IsStarted =
+			coinjoinStarted.Merge(coinjoinStopped)
```

Comment:
Here you don't merge `coinjoinCompleted` which is perfectly correct. 

It's hard in the codebase to make the distinction between a coinjoin and the coinjoin process, which was the reason of previous' PR error, and also why initial reviewers didn't catch the issue.